sampler_t.adoc

```diff
@@ -46,7 +46,7 @@ The sampler fields are described in the table below:
 
 |
 
-Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized or unnormalized values.
+Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized ([0,1[) or unnormalized ({0, size-1}) values.
```

Comment:
Wouldn't 10.5 also be a correct coordinate? If yes, then it should be `[0, size - 1]`.